scripts/setup.sh

```diff
@@ -190,8 +190,16 @@ if [ ! -d "$NODE_PATH" ]; then
 fi
 
 # Adjust path to support the default Chrome locations for Unix, Windows and Mac OS.
-if [[ $TRAVIS == 'true' ]]; then
+if [ $TRAVIS = true ]; then
   export CHROME_BIN="chromium-browser"
+elif [ $VAGRANT = true ]; then
+    # Required for headless testing in Vagrant
+    sudo apt-get install xvfb chromium-browser
+    export CHROME_BIN="chromium-browser"
+    # Used in frontend and e2e tests. Only gets set if using Vagrant VM.
```

Comment:
Maybe add a note for what this actually does, so that if someone greps XVFB_PREFIX they'll see it. (I.e. something along the lines of: performs graphical operations in memory without showing any screen output?)